README.md

```diff
@@ -83,3 +82,22 @@ There is a `Dockerfile` and an example `docker-compose.yml.example` for a produc
 After installing Docker and `docker-compose`, run `docker-compose up` in this directory to start the application.
 
 To quickly test the application locally without customizing the `docker-compose.yml`, you may run `docker-compose -f docker-compose.yml.example up` in this directory.
+
```

Comment:
Could you add `bundle exec good_job` to the README, so that people know (and are aware of) how to start the job scheduler?

We might also need to adapt the `Procfile.dev` and the docker configuration.